README.md

```diff
@@ -69,7 +68,7 @@ wagtail start myblog .
 
 Don't forget the `.` at the end of the command. It is telling Wagtail to put all of the files in the current working directory.
 
-Once all of the files are set up, you'll need to enter some commands to set up the test database and all of the migration files that Wagtail needs. You can do that with the `migrate` command.
+Once all of the files are set up, you'll need to enter some commands to set up the test database and apply all the migrations that Wagtail needs. You can do that with the `migrate` command.
```

Comment:
It’s makemigrations that makes the files, while here we use `migrate` which only changes the database.